src/main/scala/ch/openolitor/kundenportal/repositories/KundenportalReadRepositoryAsync.scala

```diff
@@ -37,9 +37,11 @@ import ch.openolitor.buchhaltung.models.RechnungId
 import ch.openolitor.buchhaltung.models.Rechnung
 import ch.openolitor.buchhaltung.models.RechnungDetail
 import ch.openolitor.arbeitseinsatz.models._
+import ch.openolitor.core.models.PersonId
 
 trait KundenportalReadRepositoryAsync {
   def getProjekt(implicit context: ExecutionContext, asyncCpContext: MultipleAsyncConnectionPoolContext): Future[Option[ProjektPublik]]
+  def getPerson(personId: PersonId)(implicit context: ExecutionContext, asyncCpContext: MultipleAsyncConnectionPoolContext): Future[Option[Person]]
```

Comment:
Is this necessary? I don't see it used anywhere. And we have the function getById